boards/atxmega-a1-xpro/include/board.h

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (C) 2021 Gerson Fernando Budke
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_atxmega-a1-xpro
+ * @{
+ *
+ * @file
+ * @brief       Board specific definitions for the ATxmegaA1 Xplained Pro board.
+ *
+ * @author      Gerson Fernando Budke <nandojve@gmail.com>
+ */
+
+#ifndef BOARD_H
+#define BOARD_H
+
+#include "cpu.h"
+#include "macros/units.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Clock configuration
+ */
+#define CLOCK_CORECLOCK     MHZ(32)
+
+/**
+ * @name   Baudrate for STDIO terminal
+ *
+ * The standard configuration for STDIO in cpu/atxmega/periph/uart.c
+ * is to use double speed.
+ *
+ * For 32MHz F_CPU following Baudrate have good error rates
+ *  115200
+ *
+ * Matches this with BAUD in Board/Makefile.include
+ *
+ * @{
+ */
+#ifndef STDIO_UART_BAUDRATE
+#define STDIO_UART_BAUDRATE (115200U)
+#endif
+/** @} */
+
+/**
+ * @name   LED pin definitions and handlers
+ * @{
+ */
+#define LED_PORT            PORTQ
+
+#define LED0_PIN            GPIO_PIN(PORT_Q, PIN3_bp)
```

Comment:
```suggestion
#define LED0_PIN            GPIO_PIN(PORT_Q, 3)
```

This is IMO easier to read - especially for RIOT users how worked previously on non-AVR platforms. If there would be indeed an MCU with `PIN<NUM>_bp != <NUM>`, this macro would have to map the second parameter to the correct value anyway.